docs/.vuepress/config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  title: 'Breadstick', // Title of the website
+  // appears in the meta tag and as a subtitle
+  description: "A simple but flexible implementation of toast style notifications for Vue.js.",
+  // Google Analytics tracking code
```

Comment:
Since we don't have analytics we can nix this for now. Or you could add it to this PR perhaps.